src/app/header/header.component.html

```diff
@@ -1,144 +1,151 @@
-<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="context && !isGettingStartedPage">
-  <div class="navbar-header">
-    <ul class="nav navbar-nav navbar-right hidden-sm hidden-md hidden-lg ">
-      <li dropdown class="pull-right dropdown" *ngIf="loggedInUser">
-        <a dropdown-open class="pull-right">
-          <div id="header_dropdownToggle_header">
+<div *ngIf="isGettingStartedPage; then showGettingStartedHeader else showHeader"></div>
+<ng-template #showGettingStartedHeader>
+  <nav class="navbar navbar-inverse navbar-pf" role="navigation">
+    <img src="../../assets/images/openshift-io_white.png" class="openshift-logo"/>
+  </nav>
+</ng-template>
+<ng-template #showHeader>
+  <nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="context">
+    <div class="navbar-header">
+      <ul class="nav navbar-nav navbar-right hidden-sm hidden-md hidden-lg ">
+        <li dropdown class="pull-right dropdown" *ngIf="loggedInUser">
+          <a dropdown-open class="pull-right">
+            <div id="header_dropdownToggle_header">
             <span class="user-icon">
               <span *ngIf="!imgLoaded" class="pficon-user"></span>
               <img (load)="onImgLoad()" src="{{ loggedInUser.attributes.imageURL }}" [ngClass]="{'show' : imgLoaded,'img-icon' : imgLoaded, 'hide' : !imgLoaded}">
             </span>
-            <span>{{loggedInUser.attributes.fullName}}</span>
-            <span class="caret"></span>
-          </div>
-        </a>
-        <ul class="dropdown-menu view-width-100">
-          <li>
-            <a [routerLink]="['/', loggedInUser.attributes.username]">
-              <span>Profile</span>
-            </a>
-          </li>
-          <li>
-            <a href="https://fabric8.io/guide/getStarted/index.html">
-              <span>Help</span>
-            </a>
-          </li>
-          <li class="divider"></li>
-          <li>
-            <a [routerLink]="['/', loggedInUser.attributes.username, '_settings']">
-              <span>Settings</span>
-            </a>
-          </li>
-          <li>
-            <a (click)="logout();">
-              <span>Log Out</span>
-            </a>
-          </li>
-        </ul>
-      </li>
-    </ul>
-  </div>
-  <div class="collapse navbar-collapse navbar-collapse-1">
-    <ul class="nav navbar-nav navbar-primary persistent-secondary navbar-left">
-      <!-- This part of the menu is dynamic based on context -->
-      <li class="context dropdown recent-items-dropdown" dropdown *ngIf="loggedInUser">
-        <a class="recent-items-toggle" dropdown-open data-toggle="tooltip" title="{{context.name}}">
-          <div id="header_dropdownToggle" class="recent-item-text">
+              <span>{{loggedInUser.attributes.fullName}}</span>
```

Comment:
When I cut and pasted the content into the template tag, this was automatically indented. I thought we would want things properly indented, but can back it out?